python_modules/dagstermill/dagstermill/cli.py

```diff
@@ -0,0 +1,93 @@
+import os
+import sys
+import click        
+import re
+
+from dagster.cli.dynamic_loader import repository_target_argument, load_target_info_from_cli_args, get_module_target_function
+
+def get_scaffolding(register_repo_str):
```

Comment:
this variable name is a bit misleading since it is an array of two strings. this is really a struct. Let's just pass in two strings with appropriate names and then a check.str_param so that we know what they are (checks are as much for documentation as anything else)